deploy/helm/elastic-agent/templates/agent/cluster-role-binding.yaml

```diff
@@ -0,0 +1,28 @@
+{{- include "elasticagent.init" $ -}}
+{{- range $presetName, $presetVal := $.Values.agent.presets -}}
+{{- $serviceAccountName := printf "agent-%s" (lower $presetName) -}}
+{{/* cluster role binding is not namespace bound so let's try to give it a unique enough name */}}
+{{- $clusterRoleName := printf "agent-%s" (cat $presetName $.Release.Name $.Release.Namespace | sha1sum) -}}
```

Comment:
I would prefer this to be more informative. The name should include the Helm Chart name, the release name and the namespace in plain text, imo. The full SHA-1 is also a bit too long.

Reply:
the issue I see with going directly with release name and namespace is that if the user has a long release name or/and a long namespace this will be a long-long cluster-role-binding name and it maybe fail as there is a max name limit.  However, If you insist no problem changing that. That said, objects managed by Helm is better not to be edited by the user so I am not entirely sure why you choose informative vs robustness here

Reply:
We can truncate it if it's too long. I'm fine with, say, truncating it at 55 chars and then adding the first 8 chars of the hash. The reason I want to be informative is that K8s clusters have many users, some of whom may simply be looking at resources without knowing which Helm releases are installed. In that case, it's much easier to tell at a glance what a given resource is for by just looking at the name.

I don't feel too strongly about this, but if you want to keep the current format, I'd do something like `elastic-agent-{short_sha}` instead. The full hash is not very readable imo.

Reply:
just pushed your proposal there, feel free to unresolve if you spot anything else

Reply:
Turns out we don't need to truncate this or include the hash. ClusterRole and ClusterRoleBinding names can be up to 255 characters. I don't think this is documented anywhere explicitly, but I tested it in a kind cluster.

So we can have a descriptive name and not risk hash collisions.

Reply:
Both release name and namespace name are capped at 63 characters, and I think preset name is reasonably small as well? So I don't think we need to truncate at all.